resources/install.rb

```diff
@@ -0,0 +1,161 @@
+unified_mode true
+
+property :version, String,
+  default: lazy { php_version },
+  description: 'Version of the php package to install.
+Defaults to platform specific numbers, see libraries/helpers.rb'
```

Comment:
Create a new property with the following:
```ruby
property :install_type, String,
  equal_to: %w(package source),
  default: 'package'
```

Then you need to wrap the code around which type of installation you're doing. Right now you have a mix of package and source.